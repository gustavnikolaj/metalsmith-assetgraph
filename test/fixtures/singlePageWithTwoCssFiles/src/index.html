<!doctype html>
<html>
    <head>
        <meta charset="utf8">
        <title>Single Page with two Stylesheets.</title>
        <link rel="stylesheet" href="stylesheet1.css">
        <link rel="stylesheet" href="stylesheet2.css">
    </head>
    <body>
        <h1>Hello World</h1>
    </body>
</html>

