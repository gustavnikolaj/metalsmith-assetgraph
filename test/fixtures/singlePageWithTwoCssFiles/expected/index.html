<!DOCTYPE html><html data-version=testFixture><head><meta charset=utf8><title>Single Page with two Stylesheets.</title><style>body{background:red}h1{color:green}</style></head><body><h1>Hello World</h1></body></html>